schema.go

```diff
@@ -106,3 +113,34 @@ func findBuildByBuilder(db *mgo.Database, builder string, buildnum int) (*LogKee
 	}
 	return build, nil
 }
+
+func UpdateFailedTest(db *mgo.Database, id bson.ObjectId) error {
+	update := bson.M{"failed": true}
+	return db.C("tests").UpdateId(id, update)
+}
+
+func GetOldTests(db *mgo.Database, now time.Time) (*[]Test, error) {
```

Comment:
I don't see why now needs to be a parameter - you can always you time.Now right?

Reply:
I feel like I had a reason at the time but who knows now :)